arm-resources/resources/2016-07-01/swagger/resources.json

```diff
@@ -0,0 +1,2449 @@
+          "description": "The tag value."
+        }
+      },
+      "description": "Resource group filter."
+    },
+    "TemplateLink": {
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "URI referencing the template."
+        },
+        "contentVersion": {
+          "type": "string",
+          "description": "If included it must match the ContentVersion in the template."
+        }
+      },
+      "required": [
+        "uri"
+      ],
+      "description": "Entity representing the reference to the template."
+    },
+    "ParametersLink": {
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "URI referencing the template."
+        },
+        "contentVersion": {
+          "type": "string",
+          "description": "If included it must match the ContentVersion in the template."
+        }
+      },
+      "required": [
+        "uri"
+      ],
+      "description": "Entity representing the reference to the deployment paramaters."
+    },
+    "DeploymentProperties": {
+      "properties": {
+        "template": {
+          "type": "object",
+          "description": "The template content. It can be a JObject or a well formed JSON string. Use only one of Template or TemplateLink."
+        },
+        "templateLink": {
+          "$ref": "#/definitions/TemplateLink",
+          "description": "The template URI. Use only one of Template or TemplateLink."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Deployment parameters. It can be a JObject or a well formed JSON string. Use only one of Parameters or ParametersLink."
+        },
+        "parametersLink": {
+          "$ref": "#/definitions/ParametersLink",
+          "description": "The parameters URI. Use only one of Parameters or ParametersLink."
+        },
+        "mode": {
+          "type": "string",
+          "description": "The deployment mode.",
+          "enum": [
+            "Incremental",
+            "Complete"
+          ],
+          "x-ms-enum": {
+            "name": "DeploymentMode",
+            "modelAsString": false
+          }
+        },
+        "debugSetting": {
+          "$ref": "#/definitions/DebugSetting",
+          "description": "The debug setting of the deployment."
+        }
+      },
+      "required": [
+        "mode"
+      ],
+      "description": "Deployment properties."
+    },
+    "DebugSetting": {
+      "properties": {
+        "detailLevel": {
+          "x-ms-client-flatten": true,
+          "type": "string",
+          "description": "The debug detail level."
+        }
+      }
+    },
+    "Deployment": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DeploymentProperties",
+          "description": "The deployment properties."
+        }
+      },
+      "description": "Deployment operation parameters."
+    },
+    "DeploymentExportResult": {
```

Comment:
Please consider adding a description element, it's essential for maintaining reference documentation.